docs/next/guides/building-and-testing/modules.md

```diff
@@ -143,129 +145,483 @@ Elements to be imported manually on demand:
+// import the PasswordEditor
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+```
+
+Afterwards you need to use the `registerEditor` method to register `PasswordEditor`:
+
+```js
+// register the editor before using it
+registerEditor(PasswordEditor);
+```
+
+Now, you can use `PasswordEditor`, the full example looks like this:
+
+```js
+import Handsontable from 'handsontable/base';
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+
+registerEditor(PasswordEditor);
+
+// use the editor as needed
+new Handsontable(container, {
+  columns: [
+    {
+      editor: 'password',
+    },
+  ]
+  // rest of the settings
+});
+```
+
+And that is all! You can use the password editor!
+
+#### Importing all editors at once
+
+You can quickly import and register all editors at once:
+
+```js
+// import all editors
+import {
+  registerEditor,
+  AutocompleteEditor,
+  BaseEditor,
+  CheckboxEditor,
+  DateEditor,
+  DropdownEditor,
+  HandsontableEditor,
+  NumericEditor,
+  PasswordEditor,
+  SelectEditor,
+  TextEditor,
+} from 'handsontable/editors';
+
+// import the `registerAllEditors()` function
+import { registerAllEditors } from 'handsontable/registry';
+
+// register all editors
+registerAllEditors();
+```
+
+## Importing renderers
+
+To use a specific renderer you need to import it alongside the registering method. For instance, let's try adding the autocomplete renderer.
+
+Start with importing the base, `autocompleteRenderer` and the `registerRenderer` method.
+
+```js
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+// import the method for registering a renderer
+// import the autocompleteRenderer
+import { registerRenderer, autocompleteRenderer } from 'handsontable/renderers';
+```
+
+Afterwards you need to use the `registerRenderer` method to register `autocompleteRenderer`:
+
+```js
+// register the renderer before using it
+registerRenderer(autocompleteRenderer);
+```
+
+Now, you can use `autocompleteRenderer`, the full example looks like this:
+
+```js
+import Handsontable from 'handsontable/base';
+import { registerRenderer, autocompleteRenderer } from 'handsontable/renderers';
+
+registerRenderer(autocompleteRenderer);
+
+// use the renderer as you need
+new Handsontable(container, {
+  columns: [
+    {
+      renderer: 'autocomplete',
+    },
+  ],
+// rest of the settings
+});
+```
 
 And that is all! You can use the autocomplete renderer!
 
+#### Importing all renderers at once
+
+You can quickly import and register all renderers at once:
+
+```js
+// import all renderers
+import {
+  registerRenderer,
+  baseRenderer,
+  autocompleteRenderer,
+  checkboxRenderer,
+  htmlRenderer,
+  numericRenderer,
+  passwordRenderer,
+  textRenderer,
+} from 'handsontable/renderers';
+
+// import the `registerAllRenderers()` function
+import { registerAllRenderers } from 'handsontable/registry';
+
+// register all renderers
+registerAllRenderers();
+```
```

Comment:
```suggestion
```js
// import the `registerAllRenderers()` function
import { registerAllRenderers } from 'handsontable/registry';

// register all renderers
registerAllRenderers();
```
```